### PR TITLE
Show display names of service classes instead of service id

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@aidenkeating change look ok

File: installer/roles/ansible-service-broker-setup/tasks/main.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 # Copy all apb images from feedhenry repo to the dockerhub_org repo - unless that is set to feedhenry
 
+# Copy all apb images from feedhenry repo to the dockerhub_org repo - unless that is set to feedhenry
```
Comment (david-martin) [inline installer/roles/ansible-service-broker-setup/tasks/main.yml]: @aidenkeating  duplicate comment here, maybe from a rebase/merge?

**Comment by aidenkeating** [comment]
@david-martin @sedroche  Would you mind taking a look? Included some screenshots of the changed sections

## Dashboard Screen
screen shot 2017-11-09 at 13 10 24

## Mobile App Screen - Available Integrations section
screen shot 2017-11-09 at 13 10 39

## Mobile Service Screen
screen shot 2017-11-09 at 13 11 02

**Comment by sedroche** [comment]
@aidenkeating I think you can use components to do this. You can create 2 components, one that displays the name and another that displays the icon.

**Comment by sedroche** [comment]
@aidenkeating Closing this for https://github.com/feedhenry/mcp-standalone/pull/238

**Pull request closed without merging**